modules/cluster-logging-loki-tech-preview.adoc

```diff
@@ -7,15 +7,15 @@
 include::snippets/technology-preview.adoc[]
 
 [id="cluster-logging-about-loki"]
-= About Loki
+== About Loki
```

Comment:
The first heading in a modules must be H1, so you need to revert this change and adjust the offset in the assembly.

All of the headings in the following modules need to be adjusted.